src/tests/Blef.GameLogic.Tests/PokerHandsHierarchy/PokerHandHierarchyTest.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Blef.GameLogic.PokerHands;
+using Blef.GameLogic.Tests.TestData;
+using NUnit.Framework;
+
+namespace Blef.GameLogic.Tests
+{
+    public class PokerHandHierarchyTest
+    {
+        [TestCaseSource(typeof(TestPokerHandsGenerator), nameof(TestPokerHandsGenerator.GeneratePokerHandsMatrix))]
+        public static void should_be_able_to_tell_which_one_pokerhand_is_stronger_than_another(int i, int j,
```

Comment:
Maybe there should be new project with only NUnit framework, so this one case can be kept there. Everything else can stay in xUnit so we have the bug regression test (#25)